src/osmf/lib/HDSPlayer.as

```diff
@@ -430,6 +432,7 @@ package
       if (_playerState == MediaPlayerState.READY || _playerState == MediaPlayerState.PAUSED 
           || _playerState == MediaPlayerState.BUFFERING)
       {
+        _playCompleted = false;
```

Comment:
Hi angie,


Because of the same reason we set  this variable
(_playCompleted)onGetCurrentTime
 .Because onGetCurrentTime event is last event we are using  _playCompleted
.but i will check again.

Thanks
Sarita Singh

On Mon, Apr 18, 2016 at 12:03 PM, angieisrani <notifications@github.com>
wrote:

> In src/osmf/lib/HDSPlayer.as
> <https://github.com/ooyala/html5-video-plugins/pull/144#discussion_r60115589>
> :
>
> > @@ -430,6 +432,7 @@ package
> >        if (_playerState == MediaPlayerState.READY || _playerState == MediaPlayerState.PAUSED
> >            || _playerState == MediaPlayerState.BUFFERING)
> >        {
> > +        _playCompleted = false;
>
> Hi Sarita,
> I think that's a good idea.
>
> However, also imagine the scenario where the movie URL is changed after a
> stream completes. At this point the _playCompleted value will be set to
> true. If someone were to query the current time it would return the
> duration of the movie instead of 0 or the initial time. This would happen
> until the user clicks play
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/html5-video-plugins/pull/144/files/643510f4579985f3bbc2bdf889d103e4333a60cb#r60115589>
>



-- 
Best Regards,
Sarita Singh